lib/mastodon/cli/accounts.rb

```diff
@@ -493,9 +491,7 @@ def reset_relationships(username)
       end
 
       if options[:followers]
-        scope = Account.where(id: ::Follow.where(target_account: account).select(:account_id))
-
-        scope.find_each do |target_account|
+        followers_for(account).find_each do |target_account|
```

Comment:
Same, `account.following.find_each` and `account.followers.find_each` result in similar but different SQL queries with identical query plans.